UIKit/Tasks/MenuViewController.swift

```diff
@@ -5,9 +5,189 @@ import UIKit
 
 /// Класс для выбора еды
 class MenuViewController: UIViewController {
+    private let guestLabelUp: UILabel = {
+        let label = UILabel()
+        label.textColor = .white
+        label.text = "Г"
+        label.font = UIFont(name: "Verdana-Bold", size: 18)
+        label.frame = CGRect(x: 336, y: 242, width: 44, height: 16)
+        return label
+    }()
+
+    private let guestAvatar: UIImageView = {
+        let image = UIImageView()
+        image.frame = CGRect(x: 320, y: 230, width: 44, height: 44)
+        image.image = UIImage(named: "g")
```

Comment:
```suggestion
        image.image = UIImage(named: "g") 
```
непонятный нейминг изображения, переименуй чтобы было понятно что за изображение